src/LatteTemplateResolver/NetteApplicationUIPresenterGlobals.php

```diff
@@ -24,18 +25,30 @@ protected function getClassGlobalVariables(ReflectionClass $reflectionClass): ar
     protected function getClassGlobalComponents(ReflectionClass $reflectionClass): array
     {
         $className = $reflectionClass->getName();
-        return $this->componentFinder->find($className, 'startup', 'beforeRender');
+        return array_merge(
```

Comment:
this is also reverted, yesterday I made find methods in Finders "multi-method" via ...